.circleci/config.yml

```diff
@@ -19,6 +19,14 @@ parameters:
     type: string
     # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-8
     default: "solbuildpackpusher/solidity-buildpack-deps@sha256:2662376fe0e1ec2d346495a19a6d64508c1048d5a7325d8600c33c343fa64a0f"
+  ubuntu-2404-docker-image:
+    type: string
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2404-1
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:5d6d27551104321a30326d6024bd4e96d9d899a97a78eb9feea364996f1d18b5"
```

Comment:
Just to let it documented, this image comes from: https://github.com/ethereum/solidity/pull/15321#issuecomment-2291244090